src/main/java/org/ow2/proactive/procci/model/occi/infrastructure/ComputeBuilder.java

```diff
@@ -44,14 +47,14 @@
 
     public ComputeBuilder() {
         this.url = "";
-        title = "";
+        title = "Compute";
         summary = "";
         architecture = null;
         cores = null;
         share = null;
         hostname = "";
         memory = null;
-        state = null;
+        state = ComputeState.ERROR;
```

Comment:
The state should be set each time, if the state is not set then there is an error.